src/PKSim.Infrastructure/Services/MoBiExportTask.cs

```diff
@@ -108,7 +108,7 @@ private void exportSimulationToFile(Simulation simulation, string moBiFile)
             throw new PKSimException(PKSimConstants.Error.CannotExportAnImportedSimulation);
 
          var configuration = _simulationConfigurationTask.CreateFor(simulation, shouldValidate: true, createAgingDataInSimulation: false);
-         var moBiSimulation = _simulationMapper.MapFrom(simulation, configuration, shouldCloneModel: false);
+         var moBiSimulation = _simulationMapper.MapFrom(simulation, configuration, shouldCloneModel: true);
```

Comment:
@Yuri05 I could not reproduce with this flag set to true. This does not modify the simulation in place but rather create another one
Check it out and let me know if the problem is fixed

Reply:
Merging. Will check with the next nightly